WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -60,6 +57,11 @@
 
               <td class="name"><%= registration.name %></td>
               <td class="countryId"><%= registration.countryId %></td>
+              <td>
+                <span data-toggle="tooltip" data-placement="left" data-container="body" title="<%= registration.created_at %>">
+                  <%= registration.created_at.to_date %>
```

Comment:
Just curious: how does this sort correctly? I figure the bootstrap-table library is only going to sort alphabetically.